src/logging/NewRelicLoggingService.js

```diff
@@ -45,6 +73,20 @@ function fixErrorLength(error) {
  * @memberof module:Logging
  */
 export default class NewRelicLoggingService {
+  constructor(options) {
+    this.config = options ? options.config : undefined;
+    /*
+        Array of explicit error message regexes. If an error message matches a regex, the error is considered
+        an *ignored* error and submitted to New Relic as a page action - not an error.
+
+        Ignored error regexes are configured per frontend application.
+        Add new error message regexes in edx-internal YAML as needed, ordered roughly by occurrence frequency.
+        The list of error regexes is represented in the .env files as a string, with multiple regexes delimited
+        by three pipes '|||'.
```

Comment:
1. Call out documentation that is specific to edx.org.
```suggestion

        For edx.org, add new error message regexes in edx-internal YAML as needed, ordered roughly by
        occurrence frequency. The list of error regexes is represented in the .env files as a string, with
        multiple regexes delimited by three pipes '|||'.
```
2. Is the three pipe delimiter true for everyone? Any parts of this that aren't edx.org specific, maybe pull out and up.
3. Consider providing an example regex.

Reply:
1. Will do.
2. Yes, that delimiter is intended to be for everyone.
3. Will do.